extensions/sql-database-projects/src/dialogs/createProjectFromDatabaseDialog.ts

```diff
@@ -152,7 +152,7 @@ export class CreateProjectFromDatabaseDialog {
 		}).component();
 
 		const connectionRow = view.modelBuilder.flexContainer().withItems([serverLabel, sourceConnectionTextBox], { flex: '0 0 auto', CSSStyles: { 'margin-right': '10px', 'margin-bottom': '-5px', 'margin-top': '-10px' } }).withLayout({ flexFlow: 'row', alignItems: 'center' }).component();
-		connectionRow.insertItem(selectConnectionButton, 2, { CSSStyles: { 'margin-right': '0px', 'margin-bottom': '-5px', 'margin-top': '-10px' } });
+		connectionRow.addItem(selectConnectionButton, { CSSStyles: { 'margin-right': '0px', 'margin-bottom': '-5px', 'margin-top': '-10px' } });
```

Comment:
why did these need to be switched from insertItem() to addItem()?

Reply:
here you are not really inserting, simply add to the end of array. addItem is easier to use.

Reply:
Yup, this isn't "fixing" anything since it'd work fine as it was, but it makes a little more sense to just add it to the end (that way you don't have to update the index if you add another item for example)